.github/workflows/ci.yml

```diff
@@ -3,8 +3,11 @@ name: Java CI
 on: [push]
 
 jobs:
-  build:
+  test:
```

Comment:
If this name changes then we'll also have to update the repo settings to prevent merge if the status of this is failed.

Reply:
ok, I'll rename back to `build`